File: oreo/submissions/accepted/oreo.cpp
#include <iostream>

using namespace std;

void solve(string S) {
    for (int i = 0; i < S.length(); i++) {
        if (S[i] == 'O') {
            cout << "[###OREO###]\n";
        } 
        else if (S[i] == 'R') {
            cout << " [--------]\n";
        }
        else if (S[i] == '&') {
            cout << '\n';
        }
    }
}

int main() {
    int T;
    cin >> T;
    for (int i = 0; i < T; i++) {
        string S;
        cin >> S;
        solve(S);
    }
}

File: oreo/submissions/accepted/oreo.py
def solve(S: str):
    for i in range(len(O)):
        if S[i] == 'O':
            print('[###OREO###]')
        elif S[i] == 'R': # ignore E since its implied E always follows R
            print(' [--------]')
        elif S[i] == '&':
            print()


def main():
    T = int(input())
    for _ in range(T):
        S = input()
        solve(S)


if __name__ == '__main__':
    main()
